zh_CN/dashboard/connections/connections.md

```diff
@@ -1,10 +1,10 @@
 # 连接管理
 
-用户可在管理连接中，查看当前连接至服务器或会话尚未过期的客户端及某一连接的详细信息和指标统计。可使用 [MQTTX](https://mqttx.app/zh) 作为客户端连接到 EMQX 进行发布订阅。也可使用[各语言提供的客户端库](https://www.emqx.io/docs/zh/v5.0/development/client.html)快速实现客户端连接至 EMQX。
+用户可在管理连接中，查看当前连接至服务器或会话尚未过期的客户端及某一连接的详细信息和指标统计。可使用 [MQTT X](https://mqttx.app/zh) 作为客户端连接到 EMQX 进行发布订阅。也可使用[各语言提供的客户端库](https://www.emqx.io/docs/zh/v5.0/development/client.html)快速实现客户端连接至 EMQX。
 
 ## 连接管理列表
 
-在连接管理列表中，用户可查看当前接入服务器或会话尚未过期的的连接列表及连接的基本信息，列表将展示各客户端连接到 EMQX 时设置的 ID 及用户名，当前的连接状态，客户端的 IP 地址，连接设置的心跳时长即连接最大空闲时长，客户端连接至 EMQX 的时间；其中客户端的 IP 地址数据由客户端的 IP 地址及客户端连接到 EMQX 时使用的客户端的端口拼接而成，还可在列表中对客户端进行踢除操作。
+在连接管理列表中，用户可查看当前接入服务器或会话尚未过期的的连接列表及连接的基本信息，列表将展示各客户端连接到 EMQX 时设置的 ID 及用户名，当前的连接状态，客户端的 IP 地址，连接设置的心跳时长即连接最大空闲时长，还有关于连接会话的信息，包括是否清除会话及会话过期间隔，客户端连接至 EMQX 的时间；其中客户端的 IP 地址数据由客户端的 IP 地址及客户端连接到 EMQX 时使用的客户端的端口拼接而成。
```

Comment:
Why is the English version divided into two paragraphs but the Chinese version is not?

Reply:
Because English is too long and too difficult to read.